Widgets/widget-MttrByType.json

```diff
@@ -0,0 +1,25 @@
+{
+  "id":"mttr-by-type",
+  "version":-1,
+  "fromVersion":"3.5.0",
+  "name":"MTTR by Type",
+  "dataType":"incidents",
+  "widgetType":"line",
+  "query":"-category:job and status:closed",
+  "isPredefined":true,
+  "dateRange":{
+    "fromDate":"0001-01-01T00:00:00Z",
+    "toDate":"0001-01-01T00:00:00Z",
+    "period":{
+      "byTo":"",
+      "byFrom":"days",
+      "toValue":null,
+      "fromValue":7,
+      "field":""
+    }
+  },
+  "params":{
+     "keys":["avg|openDuration", "count|1"],
```

Comment:
i think we dont need the count here. We need `avg|openDuration / (3600*24)` to get avg open duration in days (defiantly not in milliseconds)

Reply:
Also make sure for other widgets. Please test them